### PR TITLE
Feature join leave event

### PR DESCRIPTION
**Pull request by forfeen**
I already done join-leave event. Let's check!

### CODE REVIEW

File: Kvent/views.py [hunk 1]
```diff
@@ -73,4 +72,26 @@ def signup(request):
 def delete_event(request, event_id):
     event = Event.objects.get( pk=event_id)
     event.delete()
+    return redirect('index')
+
+@login_required
```
Comment (JirawadeeSampusri) [inline Kvent/views.py]: You should do like this @login_required(login_url='login'), Redirect to Login if user not logged

File: Kvent/views.py [hunk 2]
```diff
@@ -73,4 +72,26 @@ def signup(request):
...
+    user = request.user.id
+    try:
+        event = get_object_or_404(Event, pk=event_id)
+    except (KeyError, Event.DoesNotExist):
+        return redirect('index')
+    else:
+        event.participants.add(user)
+    return redirect('index')
+
+@login_required
```
Comment (JirawadeeSampusri) [inline Kvent/views.py]: @login_required(login_url='login'), following my suggesstion above.

**Comment by bleachjade** [comment]
You should add an initial migration file to make Travis build to pass.
image

File: Kvent/models.py [hunk 3]
```diff
@@ -71,3 +74,5 @@ def get_full(self):
     def get_photo(self):
         return self.photo
 
+    # def get_host(self):
+    #     return self.host
```
Comment (bleachjade) [inline Kvent/models.py]: delete these out

File: Kvent/models.py [hunk 4]
```diff
@@ -41,8 +40,12 @@ class Event(models.Model):
     number_people = models.IntegerField("Number of people", default=2)
     date_time = models.DateTimeField('Date and Time', default=timezone.now)
     photo = models.ImageField(upload_to='upload/', default='upload/images/no_img.png', null=True)
-    # participants = models.ManyToManyField(Info)
+    participants = models.ManyToManyField(User,null=True, blank=True, 
+                default=0)
     full = models.BooleanField(default=False)
+    # user = models.ForeignKey(User, null=True, blank=True, 
+    #               on_delete=models.CASCADE, default=0)
```
Comment (JirawadeeSampusri) [inline Kvent/models.py]: delete these comments

**Pull request merged**